### PR TITLE
blockdev_commit_firewall: add speed for commit job

### PR DESCRIPTION
**Pull request by aliang123**
Add arguements "speed" for commit job to make sure
that job in running status before/after iptables
set
    
Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2073358

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_commit_firewall.q35: PASS (271.08 s)

**Comment by aliang123** [comment]
@zhencliu @MiriamDeng, please help to review, thanks.

File: provider/job_utils.py [hunk 1]
```diff
@@ -244,6 +244,9 @@ def is_block_job_running(vm, jobid, tmo=200):
         if not job:
             LOG_JOB.warn('job %s cancelled unexpectedly', jobid)
             break
+        elif job['status'] != "running":
+            LOG_JOB.warn('job %s is not in running status', jobid)
```
Comment (zhencliu) [inline provider/job_utils.py:248]: What about use LOG_JOB.debug instead of warn, in case we get a Test Warn exception in the end.

Comment (aliang123) [reply provider/job_utils.py:248]: That's a good idea, will updated it later.

**Comment by MiriamDeng** [comment]
Test passed on ppc
 (1/1) Host_RHEL.m8.u7.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_commit_firewall: PASS (345.36 s)

**Comment by aliang123** [comment]
> Test passed on ppc, ACK (1/1) Host_RHEL.m8.u7.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_commit_firewall: PASS (345.36 s)
> 
> But just a minor reminder, it seems the below info it's too simple.
> 
Updated, thanks min.
> > id:2073358

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**